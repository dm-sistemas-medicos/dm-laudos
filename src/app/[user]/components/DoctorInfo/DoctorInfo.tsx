import users from "@/users";

import styles from "./DoctorInfo.module.scss";

const DoctorInfo = () => {
  const userInfo = users["mariana-machoski"];

  return (
    <p className={styles["doctor-info"]}>
      {userInfo.name}, {userInfo.crm}, {userInfo.rqe} - Email: {userInfo.email}{" "}
      - Instagram: {userInfo.instagram}
    </p>
  );
};

export default DoctorInfo;
