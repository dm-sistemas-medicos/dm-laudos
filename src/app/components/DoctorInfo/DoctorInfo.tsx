import userInfo from "@/users/mariana-machoski/user-info.json";

import styles from "./DoctorInfo.module.scss";

const DoctorInfo = () => (
  <p className={styles["doctor-info"]}>
    {userInfo.name}, {userInfo.crm}, {userInfo.rqe} - Email: {userInfo.email} -
    Instagram: {userInfo.instagram}
  </p>
);

export default DoctorInfo;
