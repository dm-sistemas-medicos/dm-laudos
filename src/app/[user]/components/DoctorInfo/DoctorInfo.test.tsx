import "@testing-library/jest-dom";

import { render } from "@testing-library/react";

import DoctorInfo from "./DoctorInfo";

describe("DoctorInfo component", () => {
  it("renders the component", () => {
    const { container } = render(<DoctorInfo />);

    expect(container).toMatchSnapshot();
  });
});
