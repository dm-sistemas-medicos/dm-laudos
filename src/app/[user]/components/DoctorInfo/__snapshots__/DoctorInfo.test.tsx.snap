// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DoctorInfo component renders the component 1`] = `
<div>
  <p
    class="doctor-info"
  >
    Dra. Mariana Machoski
    , 
    CRM/SC 35098
    , 
    RQE/SC 23991
     - Email: 
    marianamachoski@gmail.com
     - Instagram: 
    @marimachoski
  </p>
</div>
`;
